src/routes/profile-settings.route.js

```diff
@@ -3,9 +3,13 @@ import ProfileSettingsController from '../controllers/profile-settings.controlle
 import RouteAccessMiddleware from '../middlewares/route-access.middleware';
 import { validateToken } from '../validations/auth.validation';
 import { validateAccountProfile } from '../validations/account-profile.validation';
+import authMiddleware from '../middlewares/auth.middleware';
+import SettingsController from '../controllers/settings.controller';
+import { validateUserRole } from '../validations/role.validation';
 
 const router = express.Router();
 
 router.get('/view-profile', RouteAccessMiddleware.checkRouteAccess, validateToken, ProfileSettingsController.viewProfile);
 router.patch('/edit-profile', RouteAccessMiddleware.checkRouteAccess, validateToken, validateAccountProfile, ProfileSettingsController.editProfile);
+router.patch('/settings/roles', authMiddleware.checkUserLoggedIn, validateUserRole, authMiddleware.verifyIfUserIsAdmin, SettingsController.changeUserRole);
```

Comment:
@victor-abz Any reason why this PR contains changes not relevant to the its story?

Reply:
@bdushimi  I did this when @adesege suggested that we have API under the same prefix... and I noticed that we were having two API/users/ in index file then the team decided I can push it with this feature